extensions/azurecore/src/azurecore.d.ts

```diff
@@ -321,6 +321,10 @@ declare module 'azurecore' {
 		 * by connection providers to read/write to the same access token cache for stable connectivity.
 		 */
 		onEncryptionKeysUpdated: vscode.Event<CacheEncryptionKeys>;
+		/**
+		 * Fetches MSAL cache encryption keys currently in use, for instant access.
```

Comment:
nit: Usually not really necessary to explain why a function would be called unless it's a really special case. Just explaining what it does is enough.